llvm/lib/Target/AArch64/AArch64AsmPrinter.cpp

```diff
@@ -117,6 +118,7 @@ class AArch64AsmPrinter : public AsmPrinter {
   void LowerPATCHABLE_EVENT_CALL(const MachineInstr &MI, bool Typed);
 
   typedef std::tuple<unsigned, bool, uint32_t> HwasanMemaccessTuple;
+  std::optional<uint64_t> HwasanFixedShadowBase;
```

Comment:
We don't actually need this to be object scope, we can have this a local in `LowerHWASAN_CHECK_MEMACCESS`

Reply:
Done